src/chainparams.cpp

```diff
@@ -72,6 +72,7 @@ class CMainParams : public CChainParams {
         consensus.BIP66Height = 363725; // 00000000000000000379eaa19dce8c9b722d46ae6a57c2f1a988119488b50931
         consensus.CSVHeight = 419328; // 000000000000000004a1b34462cb8aeebd5799177f7a29cf28f2d1961716b5b5
         consensus.SegwitHeight = 481824; // 0000000000000000001c8018d9cb3b742ef25114f27563e3fc4a1902167f9893
+        consensus.TaprootHeight = 709632;
```

Comment:
In 88884ae97dee3ea915a35612531f62d1468b911b "Bury taproot deployment"

For consistency with the other buried deployments, we should have the hash listed here as a comment. My node has `0000000000000000000687bca986194dc2c1f949318629b44bb54ec0a94d8244` as block 709632.

Reply:
The block hash was not available when I opened the pull request, but I will include it on the next force push.

Reply:
Thanks, fixed